build.gradle

```diff
@@ -322,25 +344,3 @@ uploadArchives {
         System.out.println("Uploading version $version")
     }
 }
-
-//update static web docs
-task copyJavadoc(dependsOn: 'javadoc', type: Copy) {
```

Comment:
@cmnbroad Sorry for the delayed response, I was out for a while then catching up. 

I can do the addition of @documentedfeature annotations. Should that be part of this pull request or one queued up to follow right away afterward? 

We do need to set up the release system to build and publish the gatkDocs on the http://broadinstitute.github.io/picard/ website, as I don't think anyone is ready yet to have Picard docs on the GATK website only.  

Github now provides a simpler option for publishing a doc website; based on [this post](https://github.com/blog/2228-simpler-github-pages-publishing) it looks like we just need to switch a repo setting and we'll be able to simply copy the docs in a `/docs` directory. I'll edit the template to work with the picard theme. 

Am I missing anything?